Board Meeting Minutes/CCC Board Minutes 2020-03-12.md

```diff
@@ -0,0 +1,159 @@
+**Governing Board Conference Call -- 7:00am PST**
+
+**12 March 2020**
+
+1.  **Call to Order / Roll Call**
+
+    1.  **Premier Members of the CCC**
+
+        1.  Philippe Robin (Arm)\*
+
+        2.  Sergio Leunissen (Oracle)\*
+
+        3.  Mike Bursell (Red Hat)\*
+
+        4.  Dave Thaler (Microsoft)\* (TAC Chair)
```

Comment:
I don't see Stephen Walli in the list of members in any of the minutes, why is that?